packages/go_router/lib/src/router.dart

```diff
@@ -114,6 +108,9 @@ class GoRouter extends ChangeNotifier with NavigatorObserver {
   late final GoRouterDelegate _routerDelegate;
   late final GoRouteInformationProvider _routeInformationProvider;
 
+  @override
+  final BackButtonDispatcher? backButtonDispatcher;
```

Comment:
Not sure if this is correct - should GoRouter be responsible for creating this object?

Reply:
yes it needs to create a RootBackButtonDispatcher, otherwise android backbutton won't be wired up

Reply:
thanks, fixed.